frontend/app/src/pages/people/tabs/Wanted.tsx

```diff
@@ -106,7 +114,7 @@ export const Wanted = observer(() => {
       >
         {canEdit && <PostBounty widget="wanted" />}
       </div>
-      {main.createdBounties.map((w: any, i: any) => {
+      {main.createdBounties.slice(0, visibleItems).map((w: any, i: any) => {
```

Comment:
@VividhPandey003 The pagination call should be sent to the backend with `page & limit` queries, we have moved all the paginations to the backend

Reply:
So basically just passing page and query parameter to getCreatedBounties fn would do it.
And so I need to remove the slice logic as well right

Reply:
Okay. Thanks